types.ts

```diff
@@ -8,3 +8,4 @@ export enum Visibility {
 }
 
 export type Verb = string | { word: string; cw: string };
+export type Subject = string | { word: string; cw: string };
```

Comment:
Since these are identical other than the name, it'd probably be better to just rename the existing type, which would simplify the places where you've had to use `Verb | Subject`

```suggestion
export type Entry = string | { word: string; cw: string };
```